### PR TITLE
allows async interceptors

### PR DESCRIPTION
**Pull request by a14n**

### CODE REVIEW

**Comment by a14n** [comment]
ffe489b could perhaps fix #108 as I faced the same error.

File: lib/src/server/handler.dart [hunk 1]
```diff
@@ -146,7 +148,6 @@ class ServerHandler extends ServiceCall {
   }
 
   void _startStreamingRequest() {
-    _incomingSubscription.pause();
```
Comment (sigurdm) [inline lib/src/server/handler.dart]: Is this change related to the interceptors?

Comment (a14n) [reply lib/src/server/handler.dart]: It's related to async interceptor (and perhaps more as #108 reports the same error I have before the last commit ffe489b)

**Comment by a14n** [comment]
Could this PR be merged?

**Comment by sigurdm** [comment]
Yes, sorry for letting you wait.

**Comment by a14n** [comment]
no problem :) Thanks
